src/vivarium_testing_utils/fuzzy_checker.py

```diff
@@ -360,16 +362,20 @@ def _quantile_squared_error(
             actual_quantile = dist.cdf(value)
 
         if 0 < actual_quantile < 1:
-            return (
-                scipy.special.logit(actual_quantile) - scipy.special.logit(intended_quantile)
-            ) ** 2
+            return float(
+                (
+                    scipy.special.logit(actual_quantile)
+                    - scipy.special.logit(intended_quantile)
+                )
+                ** 2
+            )
         else:
             # In this case, we were so far off that the actual quantile can't even be
             # precisely calculated.
             # We return an arbitrarily large penalty to ensure this is never selected as the minimum.
-            return np.finfo(float).max
+            return float(np.finfo(float).max)
```

Comment:
I'm a bit concerned about this change -- is it always the case that the built-in Python `float` has the same precision as the numpy floats?

Reply:
I believe so. It is pretty annoying but not surprising that numpy floats and python float are treated differently with type checking

Reply:
ALso fwiw with have done the same type checking update in pseudopeople.

Reply:
It makes sense to me that they are different types; I'm slightly worried they have different behavior. But, maybe we will find that out if we run the tests :)